lib/hexpm/web/controllers/page_controller.ex

```diff
@@ -2,17 +2,18 @@ defmodule Hexpm.Web.PageController do
   use Hexpm.Web, :controller
 
   def index(conn, _params) do
-    # TODO: Add "hexpm" to queries
+    hexpm = Repository.hexpm()
+
     render(conn, "index.html", [
       container: "",
       custom_flash: true,
       hide_search: true,
       num_packages: Packages.count(),
       num_releases: Releases.count(),
-      package_top: Packages.top_downloads("all", 8),
-      package_new: Packages.recent(10),
-      releases_new: Releases.recent(10),
-      total: Packages.total_downloads
+      package_top: Packages.top_downloads(hexpm, "all", 8),
+      package_new: Packages.recent(hexpm, 10),
+      releases_new: Releases.recent(hexpm, 10),
+      total: Packages.total_downloads()
```

Comment:
I think we can show total downloads for all packages, including private ones. Or do you think we should only show for hexpm?